src/site/topic9-amortized.rst

```diff
@@ -46,6 +46,11 @@ Amortization
 * Notice how we only needed to call an :math:`O(n)` operation once every :math:`n` constant time :math:`O(1)` calls of ``push``
 * We can think of amortizing/spreading out this :math:`O(n)` work over the :math:`n` constant time calls
 
+* We think of all these parts as a whole expression
+* :math:`(n O(1) + O(n)) / (n + 1) = n O(1) / (n + 1) + O(n) / (n + 1)`
+* This tells us how much work is being done on average over the :math:`n +1` ``push`` calls
+* If we roughly divide this out, each term comes out to :math:`O(n)`
```

Comment:
Have alooksie to see if this checks out the way you were thinking.

Reply:
Looks fine to me. I'd just clarify:

> each term comes out to O(n)

then, all divided by _n_, so the end result is O(1). The way it's written now kinda makes it sound like it's O(n).

Reply:
It was a typo. Should have just been `O(1)`